### PR TITLE
Adding components and function templates

### PR DESCRIPTION
**Pull request by joser098**
*I've defined two components that will contains all charts configurations
*I've created functions which will be templates that we can use many times to make requests.
*Charts initialized with options in state which can be set

### CODE REVIEW

**Review by SebaSatelier — COMMENTED** [review]
Iooks good

**Review by SebaSatelier — DISMISSED** [review]

File: src/components/DesingSpectrumChart/DesingSpectrumChart.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import chartOptions from "./DSChartOptions";
+import { useState } from "react";
+
+const DesingSpectrumChart = () => {
+  const [ options, setOptions ] =useState(chartOptions)
+  return (
+    <>
+      <h1>Desing Spectrum</h1>
+      <HighchartsReact highcharts={Highcharts} options={chartOptions}/>
+    </>
```
Comment (SebaSatelier) [inline src/components/DesingSpectrumChart/DesingSpectrumChart.jsx:12]: Here, property options must be pairing with option of local state or i'm wrong?

**Review by SebaSatelier — CHANGES_REQUESTED** [review]
check line 11 in DesignSpectrumChart

File: src/components/DesingSpectrumChart/DesingSpectrumChart.jsx (hunk 1 above)
Comment (joser098) [reply src/components/DesingSpectrumChart/DesingSpectrumChart.jsx:12]: Yes, I'll check if it can be mutated to add the arrays (chart lines.)

**Review by SebaSatelier — DISMISSED** [review]

File: src/components/DesingSpectrumChart/DesingSpectrumChart.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import Highcharts from "highcharts";
+import HighchartsReact from "highcharts-react-official";
+import chartOptions from "./DSChartOptions";
+import { useState } from "react";
+
+const DesingSpectrumChart = () => {
+  const [ options, setOptions ] =useState(chartOptions)
+  return (
+    <>
+      <h1>Desing Spectrum</h1>
```
Comment (manduinca) [inline src/components/DesingSpectrumChart/DesingSpectrumChart.jsx:10]: Communication to users in spanish, please.

File: src/components/ExceedanceProbabilityChart/ExceedanceProbabilityChart.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { useState } from "react";
+import Highcharts from "highcharts";
+import HighchartsReact from "highcharts-react-official";
+import chartOptions from "./EPChartOptions";
+
+const ExceedanceProbabilityChart = () => {
+  const [ options, setOptions ] = useState(chartOptions)
+  return (
+    <>
+      <h1>ExceedanceProbabilityChart</h1>
```
Comment (manduinca) [inline src/components/ExceedanceProbabilityChart/ExceedanceProbabilityChart.jsx:10]: Spanish please.

**Review by manduinca — DISMISSED** [review]
I leaved some little comments.

**Review by SebaSatelier — APPROVED** [review]

**Comment by MaxiV95** [comment]
I recommend putting "ExceedanceProbabilityChart" and "DesingSpectrumChart" in zhelpers or in graphic/helpers, because they are not parent paths. Maybe src/helpers in  src/redux/helpers

**Review by FacundoPucheta — APPROVED** [review]

**Pull request closed without merging**